### PR TITLE
Documentation refactoring phase #3

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Documentation refactoring phase #3

Fixes [CCXDEV-9183](https://issues.redhat.com/browse/CCXDEV-9183)

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: docs/event_targets.md [hunk 1]
```diff
@@ -4,6 +4,121 @@ nav_order: 4
...
+Currently two event targets can be configured:
+
+1. Notification backend via Kafka topic
+1. ServiceLog via REST API
+
+
+
 ## Notification backend
 
+Events are sent into the Kafka topic provided by Notification service.
```
Comment (epapbak) [inline docs/event_targets.md:16]: You meant "Notification backend", maybe?

**Review by epapbak — APPROVED** [review]
ok!

File: README.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ CCX Notification Service
 * [Database](#database)
 * [Schema description](#schema-description)
```
Comment (juandspy) [inline README.md:23]: Update this line and move it into Description

**Review by juandspy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (juandspy) [reply README.md:23]: mb

**Pull request merged**